<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_hackWiFi</class>
 <widget class="QDialog" name="menu_hackWiFi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NASA Wi-Fi Key Cracker Saboteur v.6.66</string>
  </property>
  <widget class="QProgressBar" name="scanProgress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QListWidget" name="wifiList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>481</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="hackNearestButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hack nearest Wi-Fi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hackButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scanButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan for Wi-FI</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Available Wi-Fi List</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fullScanBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set maximally detailed search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
